L1Trigger/Phase2L1ParticleFlow/python/l1pfJetMet_cff.py

```diff
@@ -0,0 +1,38 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoMET.METProducers.pfMet_cfi import pfMet as _pfMet
+_pfMet.calculateSignificance = False
+l1PFMetCalo    = _pfMet.clone(src = "l1pfCandidates:Calo")
+l1PFMetPF      = _pfMet.clone(src = "l1pfCandidates:PF")
+l1PFMetPuppi   = _pfMet.clone(src = "l1pfCandidates:Puppi")
+
+l1PFMets = cms.Sequence(l1PFMetCalo + l1PFMetPF + l1PFMetPuppi)
+
+from RecoJets.JetProducers.ak4PFJets_cfi import ak4PFJets as _ak4PFJets
+_ak4PFJets.doAreaFastjet = False
```

Comment:
I think the problem is here @rekovic

Reply:
This is certainly a problem. Configuration objects should be customized **only** in the files that define them (and we really should clean up all the bad examples in `Configuration/StandardSequences/python`).